spec/models/saved_claim/caregivers_assistance_claim_spec.rb

```diff
@@ -3,21 +3,171 @@
 require 'rails_helper'
 
 RSpec.describe SavedClaim::CaregiversAssistanceClaim do
-  describe '.to_pdf' do
+  let(:build_claim_data_for) do
+    lambda do |form_subject, &mutations|
+      data = {
+        'fullName' => {
+          'first' => Faker::Name.first_name,
+          'last' => Faker::Name.last_name
+        },
+        'ssnOrTin' => Faker::IDNumber.valid.remove('-'),
+        'dateOfBirth' => Faker::Date.between(from: 100.years.ago, to: 18.years.ago).to_s,
+        'gender' => %w[M F].sample,
+        'address' => {
+          'street' => Faker::Address.street_address,
+          'city' => Faker::Address.city,
+          'state' => Faker::Address.state_abbr,
+          'postalCode' => Faker::Address.postcode
+        }
+      }
+
+      # Required properties for specific form_subjects
+      if form_subject == :primaryCaregiver
+        data['vetRelationship'] = %w[Father Mother Son Daughter].sample
+        data['medicaidEnrolled'] = [true, false].sample
+        data['medicareEnrolled'] = [true, false].sample
+        data['tricareEnrolled'] = [true, false].sample
+        data['champvaEnrolled'] = [true, false].sample
+      end
```

Comment:
Sample runs the risk of breaking reproducibility in a spec here. If the value really doesn't matter just set it to a default.